cmake/OpenSimConfig.cmake.in

```diff
@@ -35,6 +35,8 @@ set_and_check(@CMAKE_PROJECT_NAME@_ROOT_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@")
 # -------
 set_and_check(@CMAKE_PROJECT_NAME@_LIB_DIR "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
 
+list(APPEND @CMAKE_PROJECT_NAME@_LIB_DIR "@BTK_LIBRARY_DIRS@")
+
```

Comment:
Based on https://github.com/opensim-org/opensim-gui/pull/40 and https://github.com/opensim-org/opensim-gui/pull/40/files#diff-b8f1383df0dfd383eb189adf34c5d2abR39, it seems like `OpenSim_LIB_DIR` should contain only one path. Could we create a new variable that holds a list of paths? Like `OpenSim_DEPS_LIB_DIRS` for example? or `OpenSim_AND_DEPS_LIB_DIRS`?